StatsDemo/StatsDemo/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -66,6 +66,11 @@
       "idiom" : "ipad",
       "filename" : "Icon-76@2x.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ipad",
+      "size" : "83.5x83.5",
+      "scale" : "2x"
```

Comment:
Stupid Xcode. It's missing that icon file and it keeps regenerating that placeholder.